spring-cloud-sleuth-core/src/main/resources/META-INF/spring.factories

```diff
@@ -4,7 +4,6 @@ org.springframework.cloud.sleuth.annotation.SleuthAnnotationAutoConfiguration,\
 org.springframework.cloud.sleuth.autoconfig.TraceAutoConfiguration,\
 org.springframework.cloud.sleuth.propagation.SleuthTagPropagationAutoConfiguration,\
 org.springframework.cloud.sleuth.instrument.web.TraceHttpAutoConfiguration,\
-org.springframework.cloud.sleuth.instrument.web.TraceWebAutoConfiguration,\
```

Comment:
imported now. actually the type only does one thing which is setup skip patterns